configurations/batch/batch.ini.template

```diff
@@ -295,6 +296,13 @@ params.localFileRoot								= @TMP_DIR@/convert
 ;The remote URL, replacing params.localFileRoot
 params.remoteUrlDirectory							= /mypath
 
+[KAsyncConvertPdfLinux : KAsyncConvertWorker]
+id                                                  = 73
+friendlyName                                        = ConvertPdfLinux
+filter.jobSubTypeIn                                 = 202
```

Comment:
isn't job sub type 202 already used used for KAsyncConvertPDF

Reply:
It should be full compatible with 202 on windows encoders.